src/rpcdataacq.cpp

```diff
@@ -199,7 +199,10 @@ UniValue rpc_getblockstats(const UniValue& params, bool fHelp)
         result.pushKV("transaction", transactioncount/(double)(blockcount-emptyblockscount));
         result.pushKV("blocksizek", size_sum_blk/(double)blockcount/(double)1024);
         result.pushKV("posdiff", diff_sum/(double)poscount);
-        result.pushKV("super_spacing_hrs", (((double)l_last_time-(double)l_first_time)/(double)super_count)/3600.0);
+        if (super_count > 0)
+            result.pushKV("super_spacing_hrs", (((double)l_last_time-(double)l_first_time)/(double)super_count)/3600.0);
+        else
+            result.pushKV("super_spacing_hrs", "N/A");
```

Comment:
I think we should strive for not including "null" values in the JSON output, see https://google.github.io/styleguide/jsoncstyleguide.xml#Empty/Null_Property_Values